Makefile

```diff
@@ -59,22 +51,22 @@ clean-buildfiles:
 clean-mypyfiles:
 	rm -rf .mypy_cache/
 
+.PHONY: style-check
+style-check:
+	@flake8
+
+.PHONY: type-check
+type-check:
+	mypy sphinx/
+
```

Comment:
off topic:  I'm using this on my local (especially type-check target). The `:make` command of vim can handle the result easier. So I'd like to keep them as is.
http://vim.wikia.com/wiki/Automatically_open_the_quickfix_window_on_:make

Reply:
Oooh, neat 😄 I generally have a second pane open in `tmux` and just jump between them. Maybe I should investigate `tox` integration for `vim` too